command/agent/http_oss.go

```diff
@@ -23,14 +25,17 @@ func (s *HTTPServer) entOnly(resp http.ResponseWriter, req *http.Request) (inter
 	return nil, CodedError(501, ErrEntOnly)
 }
 
-func (s HTTPServer) auditHandler(h handlerFn) handlerFn {
+// auditHandler wraps wrap handler
```

Comment:
Not sure if these is grammatically correct - maybe?  The same applies to rest.

```suggestion
// auditHandler wraps the passed handler
```